### PR TITLE
feat(experimental): `Header` add new component

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

Closes #

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/experimental/header/header.component.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    readonly example1: TuiDocExample = {
+        HTML: import('./examples/1/index.html?raw'),
+    };
+
+    readonly example2: TuiDocExample = {
+        HTML: import('./examples/2/index.html?raw'),
+    };
+
+    readonly example3: TuiDocExample = {
+        HTML: import('./examples/2/index.html?raw'),
```
Comment (waterplea) [inline projects/demo/src/modules/experimental/header/header.component.ts:35]: Wrong example

File: projects/experimental/directives/header/header.styles.less [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        }
+
+        &:before {
+            content: '​';
+            visibility: hidden;
+        }
+    }
+
+    [tuiSubtitle],
+    [tuiCaption] {
```
Comment (waterplea) [inline projects/experimental/directives/header/header.styles.less:30]: I wish there was a way not to introduce a new selector here 🤔

File: projects/experimental/directives/header/header.styles.less [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        color: var(--tui-text-02);
```
Comment (waterplea) [inline projects/experimental/directives/header/header.styles.less:31]: In terms of lines it might be cheaper to duplicate it :)

**Review by waterplea — APPROVED** [review]

File: projects/experimental/directives/header/header.styles.less [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+        flex-grow: 2;
+    }
+
+    [tuiAccessories] {
+        display: flex;
+        align-items: center;
+        margin-inline-start: 1rem;
+
+        & > *:not(:first-child) {
+            margin-inline-start: 0.75rem;
```
[the remark this answers is not in the document]
Comment (vladimirpotekhin) [reply projects/experimental/directives/header/header.styles.less:20]: due to `:before` pseudo element

**Pull request merged**